pims/display.py

```diff
@@ -33,8 +34,9 @@
 
 
 def export_pyav(sequence, filename, rate=30, bitrate=None,
-                width=None, height=None, codec='mpeg4', format='yuv420p',
-                autoscale=True):
+                width=None, height=None, format=None, codec='mpeg4',
+                pixel_format='yuv420p', autoscale=None, quality=None,
```

Comment:
Would you note the parameter name change `format` -> `pixel_format` in the release notes?